samples/subsys/usb/webusb/src/main.c

```diff
@@ -26,20 +26,29 @@ static u8_t data_buf[64];
 
 /* WebUSB Platform Capability Descriptor */
 static const u8_t webusb_bos_descriptor[] = {
-	/* Binary Object Store descriptor */
-	0x05, 0x0F, 0x1D, 0x00, 0x01,
+	0x05, /* Descriptor size */
```

Comment:
Can you please add webusb.h header to `include/usb/class` and insert definitions for webusb descriptors there like it was done for other classes (or see ce844fab27c6ae406a51bc8a238724e4b9041422 for example)?